nautilus_trader/adapters/interactive_brokers/client/client.py

```diff
@@ -268,461 +394,242 @@ def _on_task_completed(
             if success:
                 self._log.info(success, LogColor.GREEN)
 
-    def _next_req_id(self) -> int:
+    def next_req_id(self) -> int:
         """
-        Get next available request ID.
+        Generate the next sequential request ID.
+
+        Returns
+        -------
+        int
+
         """
         new_id = self._request_id_seq
         self._request_id_seq += 1
         return new_id
 
-    def _reset(self):
-        self._stop()
-        self._client.reset()
+    def start(self) -> None:
+        """
+        Start the client.
 
-        # Start the Watchdog
-        self._watch_dog_task = self.create_task(self._run_watch_dog())
+        Returns
+        -------
+        None
 
-    def _start(self):
+        """
         self.is_ready.set()
 
-    def _resume(self):
-        self.is_ready.set()
-        self._connection_attempt_counter = 0
+    def _stop(self) -> None:
+        """
+        Stop the client and cancel running tasks.
 
-    def _degrade(self):
```

Comment:
For instance, calling `.degrade()` on the client would eventually call this `_degrade()` method (we should double check the component base class and intended usage).

Reply:
Noted - adding back the leading underscore.